spec/requests/api/braintree/failure_spec.rb

```diff
@@ -305,15 +300,8 @@
             include_examples "creates nothing"
             include_examples "processor errors"
 
-            it "creates a BraintreeCustomer that's associated with the member but has no customer_id passed from Braintree" do
-              # This is a crappy scenario. Failing a transaction when both the member and the customer are new will
-              # not create a new member, because a member gets created only on a successful action. The customer will be
-              # created locally, but will have a member_id of nil, and so the transaction will have a useless customer_id.
-              # A sub-optimal scenario to be sure, but the scenario on production is equivalent before this change (only that the
-              # customer_id in the transaction is nil, not that the customer's member_id is nil).
-              expect{ subject }.to change{ Payment::Braintree::Customer.count }.by 1
-              expect(Payment::Braintree::Customer.last.member_id).to eq nil
-              expect(Payment::Braintree::Customer.last.customer_id).to eq nil
```

Comment:
Assuming here with this change that we don't care about failing transactions? (that's the reason it was kept as it was before, iirc)